src/registers/mxcsr.rs

```diff
@@ -0,0 +1,120 @@
+//! Functions to read and write MXCSR register.
+
+#[cfg(feature = "instructions")]
+pub use self::x86_64::*;
+
+use bitflags::bitflags;
+
+bitflags! {
+    /// MXCSR register.
+    #[repr(transparent)]
+    pub struct MxCsr: u32 {
+        /// Invalid operation
+        const INVALID_OPERATION = 1 << 0;
+        /// Denormal
+        const DENORMAL = 1 << 1;
+        /// Divide-by-zero
+        const DIVIDE_BY_ZERO = 1 << 2;
+        /// Overflow
+        const OVERFLOW = 1 << 3;
+        /// Underflow
+        const UNDERFLOW = 1 << 4;
+        /// Precision
+        const PRECISION = 1 << 5;
+        /// Denormals are zeros
+        const DENORMALS_ARE_ZEROS = 1 << 6;
+        /// Invalid operation mask
+        const INVALID_OPERATION_MASK = 1 << 7;
+        /// Denormal mask
+        const DENORMAL_MASK = 1 << 8;
+        /// Divide-by-zero mask
+        const DIVIDE_BY_ZERO_MASK = 1 << 9;
+        /// Overflow mask
+        const OVERFLOW_MASK = 1 << 10;
+        /// Underflow mask
+        const UNDERFLOW_MASK = 1 << 11;
+        /// Precision mask
+        const PRECISION_MASK = 1 << 12;
+        /// Toward negative infinity
+        const ROUNDING_CONTROL_NEGATIVE = 1 << 13;
+        /// Toward positive infinity
+        ///
+        /// Set the both control registers for "toward zero".
```

Comment:
> 

I had that but did not stage that change. Coming up.

Reply:
OK I uploaded new version. My biggest struggle with QA is that I have no idea how to enforce cargo to compile "not inline_asm" flow instead of "inlin_asm" flow.

Reply:
The CI job uses `cargo build --no-default-features --features external_asm,instructions`.